dataFile

```diff
@@ -0,0 +1,278 @@
+
+Round, Team1, Team2, Score
```

Comment:
Use extensions for files.
This way it will be easier to understand what data is stored and how to read it.